recipe/meta.yaml

```diff
@@ -28,8 +28,7 @@ requirements:
     - numpy
     - peewee >=3.9.4
     - platformdirs
-    - python >=3.5
-    - requests >=1.1.0
+    - python >=3.9
```

Comment:
Need to add `bw-projects` as well.

Reply:
...and to make double-sure, that's bw-projects, but bw_processing?

Reply:
And it should be `bw-projects >=2.1.0`